hack/run-ci.sh

```diff
@@ -16,6 +16,7 @@
 
 set -e
 readonly ARTIFACTS_PATH="${ARTIFACTS}"
+export KUBEVIRT_MEMORY_SIZE=11G
```

Comment:
Removed, I was blindly guessing why CI had failed, but maybe it was a temporary thing

Reply:
For me the failure looks like something with ceph, Volumes not created, some problems like that were already fixed in the past, but maybe there is a special case that sometimes fails. I need to take a look into this if happens again.

Reply:
Looks like the memory set in config.sh does not work because it is done too late. So your addition to run-ci script was actually good and correctly worked on ci environments.